eo-runtime/src/main/eo/org/eolang/negative-infinity.eo

```diff
@@ -63,14 +63,16 @@
     x.at 0 > first!
     if. > @
       or.
-        first.as-bytes.eq (nan.as-bytes)
+        not.
+          first.eq first
         first.eq 0.0
         first.eq 0
       nan
       if.
-        first.eq ^
-        positive-infinity
+        not.
```

Comment:
@maxonfjvipon I believe it's much easier to read `if` with positive check rather than with negation:
```eo
if.
  first.eq ^
  positive-infinity
  ^
```
What do you think?

Reply:
@volodya-lombrozo sure, I'll fix it